CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+## How to contribute
+
+If you are interested in joining the project, then please review our [Prroject Wiki](https://github.com/hackforla/VRMS/wiki/Join-the-Team) and then review our [Project Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you.
+
+### Committing changes overview
+
+1. Find a ticket in the [Prioritized Backlog](https://github.com/hackforla/VRMS/projects/2) and assign yourself to the ticket, and update its status as "In progress".
+
+1. Fork the repo.
+
+1. Make a branch based on the `development` branch.
+
+1. Develop the task solution.
+
+1. Commit and push your changes to your forked repository. Please pull the recent changes from `vrms/development` before pushing your changes.
+
+1. From the `VRMS` repository, create a Pull Request from your branch into the VRMS `developnent` branch.
+
+1. Any changes will need approval and merging from a tech lead.
+
+1. Bring an update of your changes to the weekly VRMS team checkin.
```

Comment:
does this refer to discussing the update at the monday meeting, updating the github issue, or both? Once that's clear, I will propose reworded text